specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2021-03-01/containerInstance.json

```diff
@@ -399,8 +399,8 @@
           }
         ],
         "responses": {
-          "204": {
-            "description": "NoContent"
+          "201": {
```

Comment:
>"201" [](start = 10, length = 5)

For long running operations you must return 202 Accepted as referenced [here](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#call-action-post-asynchronously)